test/integration/config_test.go

```diff
@@ -0,0 +1,81 @@
+// +build integration
+
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"strings"
+	"testing"
+)
+
+func TestConfig(t *testing.T) {
+	t.Parallel()
+	p := profileName(t)
+	mk := NewMinikubeRunner(t, p, "--wait=false")
+
+	tests := []struct {
+		cmd    string
+		stdout string
+		stderr string
+	}{
+		{
+			cmd:    "config unset cpus",
+			stdout: "",
+			stderr: "",
+		},
+		{
+			cmd:    "config get cpus",
+			stdout: "",
+			stderr: "Error: specified key could not be found in config",
+		},
+		{
+			cmd:    "config set cpus 2",
+			stdout: "! These changes will take effect upon a minikube delete and then a minikube start",
+			stderr: "",
+		},
+		{
+			cmd:    "config get cpus",
+			stdout: "2",
+			stderr: "",
+		},
+		{
+			cmd:    "config unset cpus",
+			stdout: "",
+			stderr: ""},
+		{
+			cmd:    "config get cpus",
+			stdout: "",
+			stderr: "Error: specified key could not be found in config",
+		},
+	}
+
+	for _, test := range tests {
```

Comment:
rename test to tc (to be more consistent with other tests and not getting confused with go test)